open-company/open-company-web
# Comments addressability

Card: https://trello.com/c/A0xsWskz

### Review after #879 

To test:
- add some comments
- copy the url of a comment
- open that URL
- [ ] do you see the post page open?
- [ ] do you see the page scrolling down to the comment?
- [ ] is the comment highlighted?
- [ ] does the bg color of the comment fade out after 5 seconds?
- navigate back to board
- open the post again clicking on FoC
- [ ] no comment highlighed this time?
- play with comments and their urls
- now add a new post with a user B mentioned in the body
- with user B click on the notification in bottom left corner
- [ ] post page was open?
- with user B leave a comment under the post
- with user A click the notification
- [ ] comment link was open and the page scrolled down highlighting the comment?
